### DISCUSSION TITLE
Multiple filament instances/admin panels

### DISCUSSION THREAD

**Discussion by bernhardh** [filamentphp/filament#111] [Help]
Hello,

is there a way to create different filament instances/admin panels in one laravel installation? 

Let's say I have a "regular" laravel installation with everything prepared: Models, Users, Migrations, configurations, etc. Now i want to add a two differnt filament admin panels to it. One is for admins to do classic cms stuff (changing pages, creating users, clear cache, etc.) and one is for users to manage some specific models. And maybe I would need even a simple third panel for external customers.

Maybe this is achieveable with rights and roles, but the design and structure of these two admin panels would be different as well (different types of users - technical admin vs non technical users).

I could create multiple laravel installations for it, but on the other hand, there is a lot of stuff common in both panels (most of configs, most of the storage, all models, events and migrations, a lot of helper classes and other stuff).

So is there a way to achieve it in the same installation?

**Comment by iantasker** [comment]
I investigated this and it's not currently possible, I was able to get multiple panels but filtering the resources on Nav bar was the issue, all resources were being displayed on all panels. 

This was due to they way the Nav component was being rendered and what route information was available. 

Permissions and nestable navigation items should be able to solve this problem, if that feature gets added.

**Reply by danharrin** [reply]
We have permissions implemented, just need to document them.
